javascript/linkedList/linkedList/linked-list.js

```diff
@@ -4,92 +4,76 @@ const Node = require("./node");
 
 class LinkedList {
   constructor(value) {
-    this.value = value;
     this.head = null;
   }
 
   isEmpty() {
-    return this.head === null ? true : false;
```

Comment:
When you use `===` the result will be `true`/`false` already. You're essentially doing `true === true`. Instead, you can just return the result of the original comparison.